_osl/osl.env

```diff
@@ -0,0 +1,33 @@
+# Export these vars in your terminal session before running pnpm commands.
+
+# set -a
+# source osl.env
+# set +a
+
+# Use digests to update the manifests accordingly
+USE_IMAGE_DIGESTS=true
+# Won't build any images in this repo, just the manifests. The images are built within internal systems.
+KIE_TOOLS_BUILD__buildContainerImages=false
+
+# SHA required for the operator image. If you don't have it yet, just comment it.
+OSL_OPERATOR_IMAGE__buildTag=sha256:ffae74d76431b428341abfe8b8b704323261c224ff9140a2ea2782c0524f222f
+
+SONATAFLOW_OPERATOR__registry=registry.redhat.io
+SONATAFLOW_OPERATOR__account=openshift-serverless-1
+SONATAFLOW_OPERATOR__name=openshift-serverless-1-logic-rhel8-operator
+SONATAFLOW_OPERATOR__buildTag=sha256:ffae74d76431b428341abfe8b8b704323261c224ff9140a2ea2782c0524f222f
```

Comment:
could we re-use duplicated values like `OSL_OPERATOR_IMAGE__buildTag` or isn't Sonataflow and OSL operator the same?

Reply:
Since it's upstream, I don't want to change the env files in the `sonataflow-operator` package. The way we have it now in the env files, we can override it without changing the files.